### PR TITLE
review test

### PR DESCRIPTION
**Pull request by JaeHyukSim**
hello

### CODE REVIEW

**Comment by JaeHyukSim** [comment]
hmm good

File: Project3DataCollector/src/com/sist/evaluationtrend/EvaluationTrendManager.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 		try {
 
 			EvaluationTrendDAO dao = new EvaluationTrendDAO();
-			for (int k = 11; k <= 18; k++) {
+			for (int k = 16; k <= 16; k++) {
```
Comment (sbcho0630) [inline Project3DataCollector/src/com/sist/evaluationtrend/EvaluationTrendManager.java:19]: 11~18로 바꿔주세요

**Review by sbcho0630 — COMMENTED** [review]
for문 16~16을 11~18로 변경요망

File: Project3DataCollector/src/com/sist/evaluationtrend/EvaluationTrendManager.java [hunk 2]
```diff
@@ -46,9 +46,9 @@
 							String mLink = "https://movie.naver.com/" + elem.attr("href");
 
 							mLink = mLink.replace("basic", "point");
```
Comment (JaeHyukSim) [inline Project3DataCollector/src/com/sist/evaluationtrend/EvaluationTrendManager.java:48]: code review test

**Review by JaeHyukSim — COMMENTED** [review]
code review --- first try

**Pull request merged**